Update FAQ (#506)
tachiyomiorg/website: src/help/faq/README.md

Aside from network issues, it could be because the images are too big or that the decoder doesn't support that image type.

### Downloads

> #### How to download several chapters or manga simultaneously?
> 
> If you're downloading several manga, you can use different sources. In this case the app will start downloading simultaneously from up to 5 different sources.
> 
> The app doesn't allow parallel downloads from one source to prevent IP bans caused by too many requests. We know that speed is being sacrificed but it's better than not being able to access a source at all.

Reviewer: Maybe swap this around

```suggestion
#### How to download several chapters or manga simultaneously?
The app doesn't allow parallel downloads from one source to prevent IP bans caused by too many requests. We know that speed is being sacrificed but it's better than not being able to access a source at all.

However, if you're downloading several manga, and they use different sources. In this case, the app will start downloading simultaneously from up to 5 different sources.
```

We recommend you to read [this](/help/guides/reading-local-manga) guide on how to do so.

#### What do I do if I can't find the Tachiyomi folder?
If you don't see the **Tachiyomi** folder on your device, try setting download location to default and downloading a chapter of any manga so that the folder can be created.

::: guide
Set download location to default by going to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="settings_downloads"/> → **Download location** and selecting first option. Restart the app after changing download location.
:::

#### How to fix empty cover for a local manga?
Sometimes some covers for local manga aren't displayed. Follow these steps to fix it:

::: guide

> 1. Make sure you've created right folder structure. To do it, open the manga with missing cover and check if you can read chapters in the app. If not, follow [this](/help/guides/reading-local-manga) guide first.
> 1. Remember which chapters you've read and remove manga from library.
> 1. Go to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="settings_advanced"/> and tap **Clear database**. This only affects manga that aren't in your library.
> 1. Go to <Navigation item="browse"/> → **Local source** and find the manga, the cover should be fixed now. Add manga back to library, mark read chapters and add tracking if needed.

Reviewer: ```suggestion
1. Make sure you've created the right folder structure. To check it, open the manga with the missing cover and check if you can read chapters in the app. If not, follow [this](/help/guides/reading-local-manga) guide first.
1. Remember, or note down, which chapters you've read and remove manga from the library.
1. Go to <Navigation item="more"/> → <Navigation item="settings"/> → <Navigation item="settings_advanced"/> and tap **Clear database**. This only affects manga that aren't in your library.
1. Go to <Navigation item="browse"/> → **Local source** and find the manga, the cover should be fixed now. Add manga back to the library, mark read chapters and add tracking if needed.
```
